asf_heat_pump_suitability/getters/get_datasets.py

```diff
@@ -404,3 +404,20 @@ def load_df_dz_lookup(**kwargs) -> pl.DataFrame:
         **kwargs,
     )
     return df
+
+
+def load_gdf_plymouth_heat_network_zone_areas(**kwargs) -> gpd.GeoDataFrame:
```

Comment:
```suggestion
def load_gdf_plymouth_heat_network_zones(**kwargs) -> gpd.GeoDataFrame:
```
shortened for readability